bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -67,7 +66,6 @@ spec:
                           type: string
                         description: |-
                           ExtraArgs is an extra set of flags to pass to the control plane component.
-                          TODO: This is temporary and ideally we would like to switch all components to
```

Comment:
@fabriziopandini We shoudl audit our APIs for multi-line TODOs and make them single line, see: https://github.com/kubernetes-sigs/controller-tools/pull/877 (PR description)

Atm we still end up with half the TODO in the CRD, which is not great :)

Reply:
Moved to the new PR